src/main/java/gift/entity/Wish.java

```diff
@@ -1,18 +1,53 @@
 package gift.entity;
 
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "wish")
 public class Wish {
-    final private Long memberId;
-    final private Long productId;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "member_id", nullable = false)
+    private Member member;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "product_id", nullable = false)
+    private Product product;
+
+    @Column(nullable = false)
     private int quantity;
 
-    public Wish(Long memberId, Long productId, int quantity) {
-        this.memberId = memberId;
-        this.productId = productId;
+    protected Wish() {
+    }
+
+    public Wish(Member member, Product product) {
+        this.member = member;
+        this.product = product;
+        this.quantity = 1;
```

Comment:
제거될 것이기 때문에 아마 이 코멘트를 수용하지는 않으셔도 될 것 같지만 이런 경우에는 [생성자 체이닝] 방식을 활용해주는 것이 더 깔끔할 것 같아요. 😃 

```suggestion
        this(member, product, 1);
```